docs/usage/configuration.rst

```diff
@@ -165,6 +165,19 @@ Default::
 
     'aol'
 
+.. setting:: TEXTFSM_TEMPLATE_DIR
+
+TEXTFSM_TEMPLATE_DIR
+~~~~~~~~~~~~~
+
+Default path to TextFSM template directory. It is recommended to pull the Network to Code templates
+from here ``https://github.com/networktocode/ntc-ansible/tree/master/ntc_templates`` and place them inside
```

Comment:
Let's make this a link instead of a `code`.  

Change this: 
```
Network to Code templates from here ``https://github.com/networktocode/ntc-ansible/tree/master/ntc_templates``
```

To this:
```
`Network to Code templates <https://github.com/networktocode/ntc-ansible/tree/master/ntc_templates>`_

Which will render it as a hyperlink. :)